packages/cactus-odap-hermes/src/test/typescript/integration/odap/odap-api-call-with-ledger-connector.test.ts

```diff
@@ -0,0 +1,685 @@
+import http from "http";
+import fs from "fs-extra";
+import { Server as SocketIoServer } from "socket.io";
+import { AddressInfo } from "net";
+import secp256k1 from "secp256k1";
+import test, { Test } from "tape-promise/tape";
+import { v4 as uuidv4 } from "uuid";
+import { randomBytes } from "crypto";
+import { PluginObjectStoreIpfs } from "@hyperledger/cactus-plugin-object-store-ipfs";
+import { create } from "ipfs-http-client";
+import bodyParser from "body-parser";
+import express from "express";
+import { DefaultApi as ObjectStoreIpfsApi } from "@hyperledger/cactus-plugin-object-store-ipfs";
+//import { PluginRegistry } from "@hyperledger/cactus-core";
```

Comment:
@jscode017 Unless there's a specific reason to leave commented code in, please remove it if it's no longer necessary (e.g. it's dead code)